packages/styles/panel.css

```diff
@@ -3,6 +3,22 @@
   --panel-background-color: var(--white);
   --panel-heading-color: var(--gray-90);
   --panel-padding: var(--space-small);
+  --panel-content-padding: var(--space-small);
+  --panel-content-margin: 0 calc(var(--space-small) * -1);
+  --panel-header-margin: calc(var(--space-smaller) * -1)
+    calc(var(--space-small) * -1);
+  --panel-header-padding: var(--space-smaller) var(--space-small);
```

Comment:
The padding variables are tricky in part due to how the heading padding is different. Ideally I should be able to set `--panel-padding` and have it cascade everywhere where it's needed. So minimally, I think we need something like this:

```css
  --panel-padding: var(--space-small);
  --panel-content-padding: var(--panel-padding);
  --panel-header-padding: var(--panel-padding);
```

This allows for one value to be set vs 3 (and still allows for individual overrides). Alternatively, we could use a fallback value as well instead of defining them at the root which might be a little more robust:

```css
.PanelContent {
  padding: var(--panel-content-padding, --panel-padding);
}
```

I'm leaning towards the latter, what do you think?